ats/src/ats/machines.py

```diff
@@ -366,7 +367,7 @@ def launch (self, test):
                 # Priority 4 setting, set it to 1 if it is not othewise set
                 else:
                     if configuration.options.verbose:
-                        print "ATS launch setting OMP_NUM_THREADS 1 by default for as it was not specified for the test."
+                        print("ATS launch setting OMP_NUM_THREADS 1 by default for as it was not specified for the test.")
                         # print "    This should allow for threaded applications to run with non threaded tests with a single thread."
```

Comment:
This print should be updated or removed since it won't work after the switch

Reply:
Tbh I did not look that closely at each print function for this PR. Just trying to get closer to the Python 3 upgrade.